docs/src/components/ComponentDoc/ComponentDocTag.tsx

```diff
@@ -1,27 +1,82 @@
 import * as React from 'react'
 import * as _ from 'lodash'
 import { Header, Message } from 'semantic-ui-react'
+const behaviorMenuItems = require('docs/src/behaviorMenu')
 
 const headerStyle = {
   whiteSpace: 'pre-line',
 }
 
+interface DefaultBehaviorInfo {
+  name: string
+  url: string
+}
+
 class ComponentDocTag extends React.Component<any, any> {
   getTagDescription = (forTag, fromInfo) => {
     const tags = (fromInfo.docblock && fromInfo.docblock.tags) || []
     return _.result(_.find(tags, 'title', forTag), 'description')
   }
 
+  findDefaultBehaviorInfo = (fetchDefaultBehaviorInfo): DefaultBehaviorInfo => {
+    const accessibilityProperty = fetchDefaultBehaviorInfo.props.find(
```

Comment:
probably, `accessibilityDescription` would be a bit more descriptive name for this variable